dev-scripts/update-dcfs.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+set -euo pipefail
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+
+function warn() {
+    echo -e "\033[1;33mWARNING: $*\033[0m" >&2
+}
+
+function fetch_design_switcher {
+    DOC_ID=Ljph4e3sC0lHcynfXpoh9f
+
+    warn "If you made changes to the queries of the DesignSwitcher, update the nodes in the script too!"
+
+    cargo run --bin fetch --features=fetch -- \
+        --doc-id="$DOC_ID" \
+        --nodes="#SettingsView" \
+        --nodes="#FigmaDoc" \
+        --nodes="#Message" \
+        --nodes="#MessageFailed" \
+        --nodes="#LoadingSpinner" \
+        --nodes="#Checkbox" \
+        --nodes="#NodeNamesCheckbox" \
+        --nodes="#MiniMessagesCheckbox" \
+        --nodes="#ShowRecompositionCheckbox" \
+        --nodes="#UseLocalStringResCheckbox" \
```

Comment:
I am having a change to update this to UseLocalResCheckbox WIP(figma file has been updated...)
```suggestion
        --nodes="#UseLocalResCheckbox" \
```

Reply:
Thanks!